src/api/attendance.ts

```diff
@@ -125,23 +126,21 @@ const getAttendanceForMember = async ({ season, memberId }) => {
  * Fetch Attendance for all members within the season (so far)
  * Seaason is the year the game comes out (in January)
  */
-const getAttendanceForAllMembers = async (season: string) => {
+const getAttendanceStatsForAllMembers = async (season: string) => {
     const { startDate, endDate } = getStartEndOfSeason(season)
     const theEnd = isBefore(getToday(), toDate(endDate)) ? toYMD(getToday()) : endDate
-    const { data, error } = await supabase
-        .from('attendance')
-        .select('attendance_id, member_id, meeting_date, attendance, event_id, events(*)')
-        .gte('meeting_date', startDate)
-        .lte('meeting_date', theEnd)
-        .eq('events.deleted', false)
-        // doesn't work how we want it to. just returns null for an event, doesn't remove attendance record
-        .eq('events.take_attendance', true)
+    const { data, error } = await supabase.rpc('aggregate_attendance_stats', {
```

Comment:
Can you add the text of this function in the sql/README file? We need to organize that stuff better. Maybe even having actual sql files that can be run to create the functions.

Reply:
Yup, I'll put that in. I noticed that we could save SQL snippets inside of suppabase, so maybe there?

Reply:
Yes, we can, and I have saved the functions in Supabase, but I would also like to keep all the "source" in one place. Separate sql files for each function makes sense, but having one file you can run to prime the DB also makes sense.